tracer/src/Datadog.Tracer.Native/debugger_probes_instrumentation_requester.cpp

```diff
@@ -1027,13 +1014,8 @@ void DebuggerProbesInstrumentationRequester::ModuleLoadFinished_AddMetadataToMod
 
 HRESULT STDMETHODCALLTYPE DebuggerProbesInstrumentationRequester::ModuleLoadFinished(const ModuleID moduleId)
 {
-    if (!is_debugger_or_exception_debugging_enabled)
```

Comment:
iiuc, this is the hot standby code that always needs to execute so that, in case, settings now read in the managed tracer turn on DI, it will work. Could we still implement the safety kill switch though `DD_DYNAMIC_INSTRUMENTATION_MANAGED_ACTIVATION_ENABLED` ? https://docs.google.com/document/d/1KEu4rgOnB3VOTiB_uztJGwT6w4dQ6FNCryVnMQItiP8/edit?tab=t.6njjxcv5o5he#heading=h.hgsgqqfp3ubt

So that basically, if `DD_DYNAMIC_INSTRUMENTATION_MANAGED_ACTIVATION_ENABLED` is false (it should be most of the time true), we dont have any native footprint for the product

Reply:
done in badbe44391babeb0cbadb3d186bc21700d3f6de9